reactive-streams/src/test/scala/fs2/interop/reactivestreams/SubscriberSpec.scala

```diff
@@ -78,10 +78,10 @@ final class SubscriberBlackboxSpec
 
   private val counter = new AtomicInteger()
 
-  def createSubscriber(): StreamSubscriber[IO, Int] = StreamSubscriber[IO, Int].unsafeRunSync()
+  def createSubscriber(): StreamSubscriber[IO, Int] = StreamSubscriber[IO, Int](3).unsafeRunSync()
```

Comment:
should this be hard-coded to `3`? not sure the context of why this was set

Reply:
Oh, we're in a test

Reply:
we need to set it in a test :) all good :)

Reply:
oh no. that's a left over from my testing. will remove.